### ISSUE TITLE
Function strftime() is deprecated

### ISSUE THREAD

**Issue by TunglinInamin** [phpmyadmin/phpmyadmin#17624]
### Describe the bug

Function strftime() is deprecated

### To Reproduce

log in pma

### Expected behavior

not show this error

### Screenshots

![image](https://user-images.githubusercontent.com/11554678/177031754-53da21df-2c48-43b1-9173-b8535b69fd53.png)

### Server configuration

- Operating system: Centos 7.9
- Web server: nginx
- Database version: Mariadb 10.8.3
- PHP version: 8.1.7
- phpMyAdmin version: 5.2.0

### Client configuration

- Browser: Chrome
- Operating system: Windows 11

### Additional context

Deprecation Notice in ./libraries/classes/Util.php#1699
 Function strftime() is deprecated

Backtrace
```
Header.php#554: PhpMyAdmin\Util::expandUserString(string '@HTTP_HOST@ / @VSERVER@ | @PHPMYADMIN@')
Header.php#410: PhpMyAdmin\Header->getPageTitle()
ResponseRenderer.php#293: PhpMyAdmin\Header->getDisplay()
ResponseRenderer.php#411: PhpMyAdmin\ResponseRenderer->getDisplay()
PhpMyAdmin\ResponseRenderer->response()
```

**Comment by williamdes** [comment]
What version of phpMyAdmin are you using?
What PHP version are you using? 

Not filling the template makes me ask questions 😄

**Comment by TunglinInamin** [comment]
update error data

**Comment by williamdes** [comment]
Duplicate of #17249

**Comment by williamdes** [comment]
Users have reported it is fixed in the latest versions of phpMyAdmin
Could you try the latest 5.2 version in development [(phpMyAdmin 5.2+snapshot)](https://www.phpmyadmin.net/downloads/#snapshot_5.2+snapshot)?

**Comment by TunglinInamin** [comment]
I have been used it, but the error still showed.

**Comment by williamdes** [comment]
> I have been used it, but the error still showed.

@MauricioFauth do you have an idea about why it is still showed?

**Comment by LH-Eelko** [comment]
**Server configuration**
- Operating system: Centos 7.9
- Webserver: nginx/1.21.6
- Database version: MariaDB 10.3.11
- PHP version: 8.1.7
- phpMyAdmin version: 5.2.0

**Client configuration**
- Browser: Chrome Version 104.0.5112.39 (Official Build) beta (64-bit)
- Operating system: Fedora 36

Problem seems to occur in the official 5.2 release. I can confirm that it's gone in version _5.2.1-dev+20220709.47e7cfe761_ though.

**Comment by williamdes** [comment]
> I can confirm that it's gone in version 5.2.1-dev+20220709.47e7cfe761 though.

Thank you !

@TunglinInamin you have some kind of caching issue I think, maybe check you did empty `opcache` if you use it

**Comment by leuchtdiode** [comment]
I get the same error with a fresh snapshot, so seems like it is not fixed?

Version information: 5.2.1-dev+20220831.92dc9017d3
PHP version: 8.1.8

**Comment by williamdes** [comment]
> I get the same error with a fresh snapshot, so seems like it is not fixed?
> 
> Version information: 5.2.1-dev+20220831.92dc9017d3 PHP version: 8.1.8

You can be sure this is fixed, please check your setup
Can you post the error stacktrace?

**Comment by leuchtdiode** [comment]
> > I get the same error with a fresh snapshot, so seems like it is not fixed?
> > Version information: 5.2.1-dev+20220831.92dc9017d3 PHP version: 8.1.8
> 
> You can be sure this is fixed, please check your setup Can you post the error stacktrace?

Tried "all languages" instead of "only english" now compared to 2 hours ago. Same error:

PHP version: 8.1.8
Version information: 5.2.1-dev+20220831.92dc9017d3

```
Deprecation Notice in ./libraries/classes/Util.php#1699
 Function strftime() is deprecated

Backtrace

Header.php#554: PhpMyAdmin\Util::expandUserString(string '@HTTP_HOST@ / @VSERVER@ | @PHPMYADMIN@')
Header.php#410: PhpMyAdmin\Header->getPageTitle()
ResponseRenderer.php#293: PhpMyAdmin\Header->getDisplay()
ResponseRenderer.php#411: PhpMyAdmin\ResponseRenderer->getDisplay()
PhpMyAdmin\ResponseRenderer->response()
```

**Comment by MauricioFauth** [comment]
Do you have some configuration that disables the error suppression operator (`@`), like the [Xdebug's scream setting](https://xdebug.org/docs/all_settings#scream)?

**Comment by leuchtdiode** [comment]
No, there is no xdebug enabled on this server. Is there any other php.ini setting which suppresses the operator? Nothing would come to my mind.

**Comment by williamdes** [comment]
When the snapshot gets built deprecations should not be shown any more. See #17734

**Comment by leuchtdiode** [comment]
@williamdes I tested with latest 5.3 snapshot and the deprecations are gone now 👍

**Comment by williamdes** [comment]
Resolved by #17734
